contrib/lib/src/lib.rs

```diff
@@ -1,5 +1,4 @@
 #![feature(crate_visibility_modifier)]
-#![feature(never_type)]
```

Comment:
I removed it and nothing complained; I know I was surprised by one of them as well.

Reply:
FWIW I think it would be a great clippy lint; I suspect there's a number of crates that are _technically_ using features but don't actually need them.